firewall/app/main/controllers/ipController.py

```diff
@@ -0,0 +1,47 @@
+from flask import (Blueprint, request, abort)
+from flask_api import status
+from pynft import Executor
+from app.main.model.ip import IPBan
+from app.main import db
+
+import re
+
+bp = Blueprint('ip', __name__, url_prefix='/ip')
+PyNFT = Executor()
+
+regex = "^((25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\.){3}(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])$"
+
+@bp.route('/ban', methods=['POST'])
+def banIP():
+    try:
+        if (request.form.get('address') == None or request.form.get('address') == ''):
```

Comment:
should store `request.form.get('address')` in a variable

Reply:
could maybe try with `not <variable_name>` ?